com.Audentia.BlocNotes/com.Audentia.BlocNotes/DetailViewController.m

```diff
@@ -24,13 +29,45 @@ - (void)setDetailItem:(id)newDetailItem {
         [self configureView];
     }
 }
+- (IBAction)saveNote:(id)sender {
```

Comment:
If you want to save on closing the view, you could move this to viewDidDisappear. This functionality makes only sense for existing notes, though.